src/coprocessor/endpoint.rs

```diff
@@ -800,7 +792,8 @@ mod tests {
         let engine = engine::new_local_engine(TEMP_DIR, &[]).unwrap();
         let mut cfg = Config::default();
         cfg.end_point_concurrency = 1;
-        let mut end_point = Host::new(engine, worker.scheduler(), &cfg, MockCopSender::new());
+        let pd_worker = FutureWorker::new("test-pd-worker");
```

Comment:
how to check we have already sent the read statistics?

Reply:
maybe mock a pd  runner?

Reply:
seem no need now.